helm/README.md

```diff
@@ -0,0 +1,206 @@
+# pygeoapi Helm Chart
+
+This Helm chart deploys a **pygeoapi** instance, a server that publishes geospatial data using Open Geospatial Consortium (OGC) standards and the OGC API Features. This chart is designed to be flexible and easy to configure, allowing you to customize the deployment to fit your needs. 🚀
```

Comment:
Update to `publishes geospatial data using the OGC API - Features Standard from the Open Geospatial Consortium (OGC)`.